mlocal/frags/build_cli.mk

```diff
@@ -28,7 +28,7 @@ $(singularity): $(singularity_build_config) $(singularity_deps) $(singularity_SO
 singularity_INSTALL := $(DESTDIR)$(BINDIR)/singularity
 $(singularity_INSTALL): $(singularity)
 	@echo " INSTALL" $@
-	$(V)install -d $(@D)
+	$(V)umask 0022 && mkdir -p $(@D)
```

Comment:
Can we use `mkdir -m 755 -p ...` to be more explicit?

Reply:
According to this reference, without using the upper bits to `-m` it won't override the setgid.

https://www.gnu.org/software/coreutils/manual/html_node/Directory-Setuid-and-Setgid.html

Reply:
If it was so simple :), after testing the created parent directories are not affected by `-m 755` with `-p` :

`umask 0002 && mkdir -m 755 -p /tmp/install/dir1/dir2`

Reply:
arghhhhhh!! Okay :-)